shared/cluster.go

```diff
@@ -280,7 +282,7 @@ func PrintClusterState() {
 	if err != nil {
 		_ = ReturnLogError("failed to print cluster state: %w\n", err)
 	}
-	LogLevel("info", "Current cluster state:\n%s\n", res)
+	LogLevel("debug", "Current cluster state:\n%s\n", res)
```

Comment:
Should log this by default for upgrades, before upgrade and after upgrade

Reply:
true